core/src/main/java/org/apache/accumulo/core/clientImpl/NamespaceOperationsImpl.java

```diff
@@ -235,6 +265,29 @@ public Map<String,String> getConfiguration(final String namespace)
     }
   }
 
+  @Override
+  public Map<String,String> getNamespaceSpecificConfiguration(String namespace)
```

Comment:
See previous comments - is this returning configuration or properties - if just properties, then the name should reflect that.

Reply:
Thanks working on updates now, I will message when I am ready to have this reviewed again.